content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -101,24 +101,24 @@ node stability (section [Reserved memory flag](#reserved-memory-flag)).
 
 Memory Manager supports two policies. You can select a policy via a `kubelet` flag `--memory-manager-policy`:
 
-* `None` (default)
-* `Static`
+* `none` (default)
+* `static`
```

Comment:
The above two strings are defined as consts in staging/src/k8s.io/kubelet/config/v1beta1/types.go

```go
	// NoneMemoryManagerPolicy is a memory manager none policy, under the none policy
	// the memory manager will not pin containers memory of guaranteed pods
	NoneMemoryManagerPolicy = "None"
	// StaticMemoryManagerPolicy is a memory manager static policy, under the static policy
	// the memory manager will try to pin containers memory of guaranteed pods to the smallest
	// possible sub-set of NUMA nodes
	StaticMemoryManagerPolicy = "Static"

```

Reply:
Assigning @tengqm since he is already engaged on this.
/assign @tengqm